### PR TITLE
Update Paddle-Lite commit hash.

### PR DESCRIPTION
**Pull request by arogowie-intel**
### PR types
 Bug fixes

### PR changes
Others

### Describe
Upgrading used Paddle-Lite commit hash. This seem to solve problem of overlapping `framework.proto` definitions between Paddle and Paddle-Lite. This PR solves build problems of #25402

### CODE REVIEW

**Comment by arogowie-intel** [comment]
@luotao1  I've made this PR to run all CI to check whether updating Paddle-Lite solves issues with BF16 type.

**Comment by luotao1** [comment]
Could you add BF16 in `framework.proto` together?

**Comment by arogowie-intel** [comment]
> Could you add BF16 in `framework.proto` together?

Done.

**Comment by arogowie-intel** [comment]
@luotao1 I've made two experiments you asked for [here](https://github.com/PaddlePaddle/Paddle/pull/25402#issuecomment-672643072) with this branch. I used scripts from [sentiment classification](https://github.com/PaddlePaddle/models/tree/release/1.8/PaddleNLP/sentiment_classification).  All worked, no errors. Models were loaded correctly, as well inference run correct.

**Comment by arogowie-intel** [comment]
I'm attaching logs from inference:
* [branch_model_pypi_pdpd184_infer_bilstm.log](https://github.com/PaddlePaddle/Paddle/files/5104153/branch_model_pypi_pdpd184_infer_bilstm.log) This one uses model trained with this PR and inference run with PaddlePaddle installed from PyPI (1.8.4)
* [pypi_pdpd184_model_branch_infer_bilstm.log](https://github.com/PaddlePaddle/Paddle/files/5104155/pypi_pdpd184_model_branch_infer_bilstm.log) This one uses model trained with PaddlePaddle installed from PyPI (1.8.4) and inference run with this branch.

I'll attach training loss/accuracy charts tomorrow.

**Comment by arogowie-intel** [comment]
@luotao1  Here are the loss charts:
loss2
loss1

Does this satisfy your needs?

**Comment by luotao1** [comment]
Sorry for the un-detail explanation in https://github.com/PaddlePaddle/Paddle/pull/25402#issuecomment-672643072, and late relpy @arogowie-intel, since we have very hard 2.0-beta jobs currently. 

Could you use [image_classificaiton predict](https://github.com/PaddlePaddle/models/blob/develop/PaddleCV/image_classification/predict.py) for two experiments?
- Since the predict in sentiment_classification is not using C++ inference library, but predict in image_classification is.
- Although predict in image_classification is written in python, but it is[ python wrapper](https://www.paddlepaddle.org.cn/documentation/docs/zh/advanced_guide/inference_deployment/inference/python_infer_cn.html) for C++ inference library.

> I'll attach training loss/accuracy charts tomorrow.

We don't need it. You could only train one batch, we want to see if predict is normal.

@lidanqing-intel Since these two experiments is very important, if you check it OK, please give us a [how to test] report, we will let QA to verify it. Thanks very much!

**Comment by arogowie-intel** [comment]
@luotao1  Thanks for your clarifications. I'm attaching script with steps to reproduce and logs from running prediction.
[infer_branch_model_pypi.log](https://github.com/PaddlePaddle/Paddle/files/5117518/infer_branch_model_pypi.log)
[infer_pypi_model_branch.log](https://github.com/PaddlePaddle/Paddle/files/5117519/infer_pypi_model_branch.log)
[repro_steps.txt](https://github.com/PaddlePaddle/Paddle/files/5117527/repro_steps.txt)

**Comment by luotao1** [comment]
LGTM @arogowie-intel I will let QA to test.

**Comment by OliverLPH** [comment]
Hi @arogowie-intel @luotao1 
I have tried steps you provided, the status of running prediction looks correct.

**Review by wojtuss — APPROVED** [review]
LGTM

**Review by luotao1 — APPROVED** [review]

**Review by Superjomn — APPROVED** [review]
LGTM

Our QA has checked the enum forward and backward compatibility.
